docs/tables/wiz_vulnerability_finding.md

```diff
@@ -0,0 +1,90 @@
+# Table: wiz_vulnerability_finding
+
+The `wiz_vulnerability_finding` table can be used to list all vulnerabilities that Wiz scanner detected on the cloud resources.
+
+**Note**: The table can limit a large dataset; which can increase the query execution time. It is recommended that queries to this table should include (usually in the `where` clause) at least one of these columns:
+
+- `detection_method`
+- `has_cisa_kev_exploit`
+- `has_exploit`
+- `severity`
+- `status`
+- `vulnerability_external_id`
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  vulnerable_asset ->> 'name' as resource_name,
+  name as vulnerability,
+  detailed_name,
+  severity,
+  fixed_version,
+  detection_method,
+  has_exploit,
+  vulnerable_asset ->> 'subscriptionId' as subscription
+from
+  wiz_vulnerability_finding;
+```
+
+### List resources with high-severity known public exploit vulnerabilities
+
+```sql
+select
+  vulnerable_asset ->> 'name' as resource_name,
+  name as vulnerability,
+  detailed_name,
+  severity,
+  fixed_version,
+  detection_method,
+  has_exploit,
+  vulnerable_asset ->> 'subscriptionId' as subscription
+from
+  wiz_vulnerability_finding
+where
+  has_exploit
+  and severity = 'HIGH';
+```
+
+### List high severity findings which are fixed
```

Comment:
I think it is better to add a query about high severity findings that are not fixed